### DISCUSSION TITLE
Info Request - All possible domains for media_content_id:

### DISCUSSION THREAD

**Discussion by rxritalin** [music-assistant/support#2773] [Q&A] [answered]
I am creating an entity that displays the correct Music Provider icon and title in the lovelace UI. I already have all the SVG files for the providers ready to go. I just dont know all the possible domains to make it work. 
 I need some help with finding all the possible domains for the media_content_id: attribute. 
I obviously do not have every Music Provider. 
Can one of the devs provide a list for all of these domains. 

For instance I right now the output is media_contact_id: spotify://track/7fmJGzyvOcbh6UANsH8Cp6
I am using `{{ state_attr('media_player.squeeze_rxstereo',  'media_content_id').split(':')[0]|title }}` to remove everything but the domain, which leaves me with just 'Spotify'. 
I know Youtube Music has a domain of 'ytmusic', Qobuz is 'qobuz', and it seems any and all radio stations simply list as 'library', but I dont know the rest. 

Apple Music - ?
Deezer - ?
Filesystem (local disk) - ?
Filesystem (remote share) - ?
Jellyfin Media Server Library -?
Open Subsoncin Media Server Library - ?
Plex Media Server Library - ?
Qobuz - qobuz
Soundclound - ?
Spotify - spotify
Tidal - ?
Tune-In Radio - ?
Youtube Music - ytmusic
Music Assistant - ?
RadioBrowser - library

**Comment by OzGav** [comment, marked as answer]
You can find the domains in the `manifest.json` files in the various folders here https://github.com/music-assistant/server/tree/main/music_assistant/server/providers 

Having said that you might actually need the URI root which I see is sometimes different when I use `mass.search`. For example:

TuneIn - library
Filesystem - library

Looks like Tidal might have the full provider identified as I see `tidal--bCA86LLE`

Where you have "Music Assistant" I assume you mean a builtin playlist? If so the ID will change depending on the origin of the track.
